qa/wildfly/build.gradle

```diff
@@ -73,177 +45,52 @@ dependencies {
-      }
-    }
+dockerCompose {
+  if ('default'.equalsIgnoreCase(System.getProperty('tests.distribution', 'default'))) {
+    useComposeFiles = ['docker-compose.yml']
+  } else {
+    useComposeFiles = ['docker-compose-oss.yml']
   }
 }
 
-task configureClient(type: LoggedExec) {
-  dependsOn startWildfly
-  // we skip these tests on Windows so we do not need to worry about compatibility here
-  commandLine "${wildflyInstall}/bin/jboss-cli.sh",
-    "--controller=localhost:${-> managementPort}",
-    "--connect",
-    "--command=/system-property=elasticsearch.properties:add(value=\${jboss.server.config.dir}/elasticsearch.properties)"
-}
-
-task stopWildfly(type: LoggedExec) {
-  // we skip these tests on Windows so we do not need to worry about CLI compatibility here
-  commandLine "${wildflyInstall}/bin/jboss-cli.sh", "--controller=localhost:${-> managementPort}", "--connect", "command=shutdown"
-}
-
-if (!Os.isFamily(Os.FAMILY_WINDOWS)) {
-  integTestRunner.dependsOn(configureClient)
-  final TaskExecutionAdapter logDumpListener = new TaskExecutionAdapter() {
-    @Override
-    void afterExecute(final Task task, final TaskState state) {
-      if (task != startWildfly && task != integTestRunner) {
-        // we might have been called from a parallel, unrelated task
-        return
-      }
-      if (state.failure != null) {
-        final File logFile = new File(wildflyInstall, "standalone/log/server.log")
-        println("\nWildfly server log (from ${logFile}):")
-        println('-----------------------------------------')
-        final Stream<String> stream = Files.lines(logFile.toPath(), StandardCharsets.UTF_8)
-        try {
-          for (String line : stream) {
-            println(line)
-          }
-        } finally {
-          stream.close()
-        }
-        println('=========================================')
-      }
-    }
-  }
-  startWildfly.doFirst {
-    project.gradle.addListener(logDumpListener)
-  }
-  integTestRunner.doFirst {
-    project.gradle.addListener(logDumpListener)
-  }
-  integTestRunner.doLast {
-    project.gradle.removeListener(logDumpListener)
-  }
-  startWildfly.doLast {
-    project.gradle.removeListener(logDumpListener)
-  }
-  integTestRunner.finalizedBy(stopWildfly)
-} else {
-  integTest.enabled = false
-  testingConventions.enabled = false
+task integTest(type: Test) {
+  outputs.doNotCacheIf('Build cache is disabled for Docker tests') { true }
+  maxParallelForks = '1'
+  include '**/*IT.class'
 }
 
-check.dependsOn(integTest)
+check.dependsOn integTest
 
 test.enabled = false
-
 dependencyLicenses.enabled = false
 dependenciesInfo.enabled = false
-
 thirdPartyAudit.enabled = false
 
-
 testingConventions {
   naming.clear()
-  // We only have one "special" integration test here to connect to wildfly
   naming {
     IT {
-      baseClass 'org.apache.lucene.util.LuceneTestCase'
+      baseClass "org.apache.lucene.util.LuceneTestCase"
```

Comment:
Why change from a `String` to a `GString`?

Reply:
What actually happened here is that a stripped a ton of stuff out of this file, then added stuff back in when I discovered that I needed it. This was such a piece, which is why it looks like all I did was change the quotes.